cmd/neutree-cli/app/cmd/launch/core.go

```diff
@@ -87,6 +88,19 @@ func installNeutreeCoreSingleNodeByDocker(exector command.Executor, options neut
 		return errors.Wrap(err, "prepare neutree core launch config failed")
 	}
 
+	if options.dryRun {
```

Comment:
Dry run doesn't seem like a common launch scenario parameter.
If there is a debugging requirement, can we have a separate print components command?

Reply:
I think `--dry-run` is a common parameter used to check whether the configuration is correct when changes need to be made. 
this parameter is not limited to launch,  but can also be extended to down, etc.